src/include/connection.h

```diff
@@ -501,6 +501,7 @@ struct __wt_connection_impl {
     wt_shared volatile uint64_t cache_size; /* Cache size (either statically
                                      configured or the current size
                                      within a cache pool). */
+    WT_EVICT *evict;
```

Comment:
I wonder if you considered including the evict structure right in the parent rather than making it a separate allocation? What would be the trade-offs? Some pros and cons?

Reply:
evict struct in the parent to cache?